homeassistant/core.py

```diff
@@ -248,12 +248,16 @@ def sleep_wait():
 
         def notify_when_done():
             """Notify event loop when pool done."""
+            count = 0
```

Comment:
This is reducing the flakiness of the tests by running `block_till_done()` twice.